Frameworks/include/CppUtils.h

```diff
@@ -36,7 +36,7 @@ std::basic_ostream<T>& operator<<(std::basic_ostream<T>& os, const CFRange& rang
 
 #pragma region CGPoint
 inline bool operator==(const CGPoint& lhs, const CGPoint& rhs) {
-    return ((std::abs(lhs.x - rhs.x) < 0.00001) && (std::abs(lhs.y - rhs.y) < 0.00001));
+    return lhs.x == rhs.x && lhs.y == rhs.y;
```

Comment:
This will almost certainly make tests fail on ARM. We had these set up for a reason. What's up?